user/src/main/java/org/dicadeveloper/weplantaforest/projects/Price.java

```diff
@@ -37,6 +41,7 @@
     private ScontoType scontoType = ScontoType.NONE;
 
     @Column(name = "_amount", precision = 7, scale = 2)
+    @JsonView({Views.Project.class})

```

Comment:
I update your comment to apply formatting. github allows to put preformatted text in 

\```

to achieve this.